examples/qtsupport/chatwidget.hpp

```diff
@@ -11,6 +11,13 @@ CAF_PUSH_WARNINGS
 #include <QTextEdit>
 CAF_POP_WARNINGS
 
+CAF_BEGIN_TYPE_ID_BLOCK(qt_support, first_custom_type_id)
```

Comment:
```suggestion
CAF_BEGIN_TYPE_ID_BLOCK(qtsupport, first_custom_type_id)
```

I find it confusing to introduce an underscore here if the example directory does not.